### DISCUSSION TITLE
Swiss or hardware failure?

### DISCUSSION THREAD

**Discussion by Rolygon1985** [emukidid/swiss-gc#797] [General]
I'm using v0.6r1443 with Picoboot. Many games run fine but Rogue leader always crashes early into the first mission. I confirmed the rip is good. The U.S version of F-zero also has problems, but the EU version in perfect. The gamecube was not in good condition when I got it so I'm wondering if maybe it's just hardware failure as I haven't seen any problems with these games online. Just strange that so many other games run fine.

**Comment by emukidid** [comment]
Can you try these via disc without Swiss involved at all? Can you describe the crash in Rogue Leader and also the "problems" you've mentioned about F-Zero GX?

**Reply by Rolygon1985** [reply]
I can't currently test either with a disc, but I have a Rogue Leader disc coming soon. 

F-Zero US version always reboots to the the Sega logo screen right before a race and sometimes before I can even press start at the title screen. 

Rogue leader will freeze with music still playing early on in the first mission., but never during the long cutscene before it starts. 

I should mention I'm also using the SD2SP2 Pro.  I'm also using a 3rd party Tomee PSU, if that could make a difference.  Tried using Nkit versions too with the same results.
